src/test/java/com/guysfromusa/carsgame/control/round/SameTypeOnePerCarGameRoundSelectorTest.java

```diff
@@ -0,0 +1,78 @@
+package com.guysfromusa.carsgame.control.round;
+
+import com.guysfromusa.carsgame.control.AddCarToGameCommand;
+import com.guysfromusa.carsgame.control.Command;
+import com.guysfromusa.carsgame.control.MoveCommand;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import java.util.LinkedList;
+import java.util.Optional;
+
+import static com.guysfromusa.carsgame.control.MessageType.ADD_CAR_TO_GAME;
+import static com.guysfromusa.carsgame.control.MessageType.MOVE;
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Created by Robert Mycek, 2018-05-05
+ */
+@RunWith(MockitoJUnitRunner.class)
+public class SameTypeOnePerCarGameRoundSelectorTest {
+
+    @InjectMocks
```

Comment:
do you need this ? I don't see any mocks

Reply:
but I do not have to write:

@Before
public void setUp() {
   this.service = new SameTypeOnePerCarGameRoundSelector();
}